app/views/devise/shared/_links.html.haml

```diff
@@ -9,3 +9,7 @@
 - if devise_mapping.lockable? && resource_class.unlock_strategy_enabled?(:email) && controller_name != 'unlocks'
   = link_to "Didn't receive unlock instructions?", new_unlock_path(resource_name)
   %br
+
+- if devise_mapping.ichain_registerable?
+  = link_to "Sign up", new_ichain_registration_path(resource_name)
```

Comment:
Yes, true, that's what happens in production.
The correct path though is `new_user_ichain_registration_path`

@jmks Where did you find the path you are using `new_ichain_registration_path` ?